mxcube3/routes/Login.py

```diff
@@ -125,7 +125,6 @@ def loginInfo():
 
     login_info = login_info["loginRes"] if login_info is not None else {}
     login_info = limsutils.convert_to_dict(login_info)
-    limsutils.update_mxcube_session(login_id, login_info)
```

Comment:
select_proposal is called in loginInfo, I think this is the same solution as the other method, just taking into account user/proposal login. Did you try running?

No need to accept per se if you do not agree. I can modify on this pr.